app/views/shop/products/_applied_filters_feedback.haml

```diff
@@ -0,0 +1,9 @@
+%span{ "ng-show" => "query && ( appliedPropertiesList() || appliedTaxonsList() )" }
```

Comment:
I am not sure about this logic, if no query, it will still show all the elements below...

Reply:
Yes, we list selected filters even when there's no query. I'll make a gift to show the behaviour.

Reply:
ok, just asking :+1:

Reply:
thanks for the gif!